### PR TITLE
Change defaults for imuplacer

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #2190 

### Brief summary of changes
Changed default values for properties of IMUPlacer to not transform or apply heading correction, updated test files to specify the old defaults and beefed up error checking to account for potential inconsistencies

### Testing I've completed
Ran tests
### Looking for feedback on...
Text of error messages

### CHANGELOG.md (choose one)

- no need to update because internal

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2640)

### CODE REVIEW

File: Applications/opensense/test/imuPlacerFaceNegX.xml [hunk 1]
```diff
@@ -3,10 +3,12 @@
 	<IMUPlacer>
 		<!--Name/path to the xml .osim file used to load a model to be calibrated.-->
 		<model_file>subject07.osim</model_file>
-		<!--The label of the base IMU in the orientations_file used to account for the heading difference between the sensor data and the forward direction of the model. Leave blank if no heading correction is applied.-->
+		<!--The label of the base IMU in the orientation_file_for_calibration used to account for the heading difference between the sensor data and the forward direction of the model. Leave blank if no heading correction is applied.-->
```
Comment (jenhicks) [inline Applications/opensense/test/imuPlacerFaceNegX.xml]: "Leave blank if no heading correction is desired"

File: OpenSim/Simulation/OpenSense/IMUPlacer.h [hunk 2]
```diff
@@ -60,17 +59,15 @@ class OSIMSIMULATION_API IMUPlacer : public Object {
     OpenSim_DECLARE_PROPERTY(base_imu_label, std::string,
             "The label of the base IMU in the orientation_file_for_calibration used to account "
             "for the heading difference between the sensor data and the forward "
-            "direction of the model. Leave blank if no heading correction is applied."
-            "Default to pelvis_imu.");
+            "direction of the model. Leave blank if no heading correction is applied.");
```
Comment (jenhicks) [inline OpenSim/Simulation/OpenSense/IMUPlacer.h]: "Leave blank if no heading correction is desired."

Comment (jenhicks) [reply OpenSim/Simulation/OpenSense/IMUPlacer.h]: Default is no heading correction applied.

File: OpenSim/Simulation/OpenSense/IMUPlacer.h [hunk 3]
```diff
@@ -60,17 +59,15 @@ class OSIMSIMULATION_API IMUPlacer : public Object {
...
 
     OpenSim_DECLARE_PROPERTY(base_heading_axis, std::string,
             "The axis of the base IMU that corresponds to its heading "
-            "direction. Options are 'x', '-x', 'y', '-y', 'z' or '-z'."
-            "Default to z.");
+            "direction. Options are 'x', '-x', 'y', '-y', 'z' or '-z'.");
```
Comment (jenhicks) [inline OpenSim/Simulation/OpenSense/IMUPlacer.h]: I would add the same info as for the base_imu_label

**Comment by jimmyDunne** [comment]
I ran the code locally and tested the behavior when a label is added or not. It seems to work as expected-- when no label is given, heading correction is not performed. 

I pushed a simple name update for an example file and a updated xml to this branch since I was on it :)

**Comment by aymanhab** [comment]
The confluence page claims visualizer window will show up on Calibration, that's not the default behavior as of now, should we make it default for any/all pathways or just say load the model in OpenSim GUI?

**Pull request merged**